### ISSUE TITLE
HDR video becomes completely dark on Macbook pro 2022

### ISSUE THREAD

**Issue by SongTonyLi** [iina/iina#4216]
**System and IINA version:**

- macOS  Ventura 13.2
- IINA 
1.3.1
**Expected behavior:**
IINA should be able to play HDR well.

**Actual behavior:**
No matter in full-screen mode or not, the HDR video becomes completely dark and I could hardly watch anything.

<details>
<summary><b>Crash report:</b></summary>

```
```
</details>

<br>

<details>
<summary><b>mpv log:</b></summary>

```
```
</details>

<br>

**Steps to reproduce:**
Open an HDR film with format mkv, and the film is just displayed dark. If I turn off the HDR option, the film becomes bright again, but I can tell that the video quality is not HDR without HDR option on.

- [ ] MPV does not have this problem.

**How often does this happen?**
always, as long as I play an hdr video.

**Comment by low-batt** [comment]
The MacBook Pro with the M2 chip? If so, then this is a duplicate of issue #4214. See my response to that issue.

**Comment by stanleystfs** [comment]
I had a similar issue on macbook pro m1 but after updating to the latest version of IINA, it went away.

**Comment by low-batt** [comment]
Both the M1 and M2 Macs exhibit a similar HDR dimming problem. But the behavior between the two chips differ. The problem was not present on the M1 under macOS Monterey. The problem appeared with macOS Ventura. It only happened in full screen mode and only once the OSC disappeared. A really horrible workaround was added in IINA 1.3.1 that adds a tiny transparent view to the displayed content so there is always more than one view being displayed. So far this ugly workaround has been effective on the M1 to suppress the problem.

The situation is different on the M2. The problem was noticed on the M2 MacBook Air under macOS Monterey. It is not sensitive to the presence of the OSC. The workaround is not effective.

Because the M1 Macs were working fine before macOS Ventura the expectation is that this is a regression in macOS. However until the actual root cause is identified we can't say for sure what is causing this.

**Comment by low-batt** [comment]
Marking this as a duplicate of #4214
